src/async.action.js

```diff
@@ -37,14 +37,14 @@ const _dedupedPromises = {};
  * The optional 'options' parameter gives you more control:
  *   * identifier can be used to disambiguate two instances of the same action.
  */
-export const createAsyncAction = <Action: SimpleAction, Payload>(
-  action: Action,
-  operation: AsyncThunk,
+export const createAsyncAction = <AAction: AsyncAction<*, *>>(
```

Comment:
should this be

```suggestion
export const createAsyncAction = <AAction: AsyncAction<SimpleAction, *>>(
```

for backwards compat?

it brings the flow errors from 114 -> 40 on mobile 😅 

> Cannot call createAsyncAction because object literal [1] is incompatible with empty [2].

Reply:
wait are you guys actually using the old generic on mobile?

Reply:
We're not typing the generics explicitly at the callsite if that's what you're asking. We don't import `SimpleAction` either.

It seems flow is doing some unexpected inference with that `*`

Here's the full issue:

<img width="846" alt="Screen Shot 2019-10-23 at 7 09 16 AM" src="https://user-images.githubusercontent.com/1047502/67404676-bce19f00-f581-11e9-8c71-6b3a16914865.png">

Reply:
I was hoping to get away with `feat:` - technically changing the generic args to `createAsyncAction` is an API change, but I audited our code bases and no-one is actually using the generic arg anyway.